jenkins/init.sls

```diff
@@ -1,5 +1,12 @@
 {% from "jenkins/map.jinja" import jenkins with context %}
 
+make_sure_global_dependencies_are_installed:
+  pkg.installed:
+    - pkgs:
+      - curl
+      - git
+      - java-1.8.0-openjdk
```

Comment:
I suggest they stay on the jinja file as we can override on the pillar if needed... I couldn't install java-1.8.0-openjdk for example.